### DISCUSSION TITLE
Abaqus inp file

### DISCUSSION THREAD

**Discussion by khriama** [neperfepx/neper#296] [Q&A] [answered]
Hi,
First of all, thanks for developing such a great tool.
I am new to using Neper.
I am trying to generate an Abaqus inp file from neper.
These are the exact commands I am using:
```
neper -T -n 100 -id 1 -reg 1
neper -M n100-id1.tess -format 'inp' -faset 'faces'
```

It worked fine and I get the inp file, my problem is: importing this inp file to Abaqus (vesion 2021). I could not create my model in Abaqus and Abaqus gave me this warning (attached).

Also, I used these commands, I get the same results:
```
neper -T -n 2
neper -M n2-id1.tess -dim all -format inp
```

Do you have any idea about this problem?
Thank you and I am looking forward to your answer.
Best regards,
Manhal 
[My files.zip](https://github.com/rquey/neper/files/7232239/My.files.zip)

**Comment by khriama** [comment, marked as answer]
Hi Romain,
Again, thank you very much for your reply and help.
 I updated my Neper with version 4.2   and get the same problem.
[My files.zip](https://github.com/rquey/neper/files/7244209/My.files.zip)

**Reply by smallrain1996** [reply]
As of now -faset 'faces' is not suitable
Please use -faset none

**Reply by rquey** [reply]
@smallrain1996 is right. Use `-faset none` with inp output, or the last development version (4.1.2-11). This will be fixed in the next official version.
